Formula/agda.rb

```diff
@@ -42,10 +31,12 @@ class Agda < Formula
     end
   end
 
+  deprecated_option "without-malonzo" => "without-ghc"
+
   option "without-stdlib", "Don't install the Agda standard library"
-  option "without-malonzo", "Disable the MAlonzo backend"
+  option "without-ghc", "Disable the GHC backend"
 
-  if build.with? "malonzo"
+  if build.with? "ghc"
```

Comment:
```
  option "without-ghc", "Disable the GHC backend"

  if build.with? "ghc"
    depends_on "ghc"
  else
    depends_on "ghc" => :build
  end
```
I think can instead be

```
depends_on "ghc" => :recommended
depends_on "ghc" => :build if build.without? "ghc"
```
which automatically gives you
```
--without-ghc
	Build without ghc support
```
If the other message is preferred for some reason, you can make it
```
option "without-ghc", "Disable the GHC backend"
depends_on "ghc" => :recommended
depends_on "ghc" => :build if build.without? "ghc"
```